### PR TITLE
I93 real auth

### LINKED ISSUE #93: Implement REAL authentication on server
Requires #94 #97

### PR DESCRIPTION
**Pull request by oktay-sen**
Closes #93

### CODE REVIEW

**Comment by freddiejbawden** [comment]
I think we should move the block comment in auth.js to a Markdown file, so we can render it

File: server/auth.js [hunk 1]
```diff
@@ -0,0 +1,70 @@
+const model = require('./model')
+
+/*
```
Comment (freddiejbawden) [inline server/auth.js]: We should put this in a MD file

Comment (freddiejbawden) [inline server/auth.js]: We should put this in an MD file

Comment (oktay-sen) [reply server/auth.js]: Moved it into server/README.md

**Pull request merged**